aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -153,31 +153,31 @@ You do not have to sign in right now to continue with this tutorial. If you do n
 
 At this point, either sign in with your Microsoft account or select **Not Now** in the lower right corner.
 
-![Sign In](getting-started/_static/image7.png)
+![Screenshot of the Windows Azure sign in dialog with the Microsoft account and organizational account sign in options.](getting-started/_static/image7.png)
 
 To begin, you'll create a blank website and add a page. In a later tutorial in this set you'll play with one of the built-in website templates.
 
 In the start window, click **New**.
 
-![WebMatrix startup screen](getting-started/_static/image8.png)
+![Screenshot of the Web Matrix Quick Start window showing the New icon highlighted with a red rectangle.](getting-started/_static/image8.png)
```

Comment:
Don't change valid alt-text

Reply:
Alt text was modified to reflect Microsoft’s accessibility guidelines for alt text per https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main.